pkg/lintcontext/parse_yaml.go

```diff
@@ -34,6 +37,14 @@ var (
 	decoder      = serializer.NewCodecFactory(clientSchema).UniversalDeserializer()
```

Comment:
I think you may have to move this call into the `init` function; otherwise, the `init` will get executed _after_ this line, and I don't think the update `schema` will be reflected in the `decoder`. Or were you able to confirm that it successfully worked on a `DeploymentConfig` after this change?

Reply:
thanks @viswajithiii 
`decoder` defined as global, so it's within `init()` scope. Yes I tested code as well 
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/16170560/113859949-cfd54700-97e8-11eb-874c-5dc7df77bf6d.png">